qnv.py

```diff
@@ -1,37 +1,24 @@
 import numpy as np
 import math
 
-def dot1(v1,v2):
-	v3 = v1.reshape((3,))
-	v4 = v2.reshape((3,))
-	v5 = np.dot(v3,v4)
-		
+def dot(v1,v2): #returns dot product of 2 row vectors
+
+	v5 = np.dot(v1,v2)		
 	return v5
 
-def cross1(v1,v2):
-	#for cross product of 2 column vectors
+def cross(v1,v2): #returns cross product of 2 row vectors

```

Comment:
Why is this needed?

Reply:
For quatMultiply we need to do cross product of 2 vectors.